api/interview_routes.py

```
@@ -741,23 +916,81 @@ async def get_profile_card(session_id: str):
         situational_report=situational_report
     )
 
-    return profile_card
+    # 백엔드에 매칭 벡터 POST
+    backend_client = get_backend_client()
+    try:
+        backend_response = await backend_client.post_matching_vectors(
+            vectors_data=result["vectors"],
+            access_token=request.access_token
+        )
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)